src/utils/config.ts

```diff
@@ -21,6 +21,18 @@ const parseAssert = (name: string, condition: any, message: string) => {
 };
 
 const configParsers = {
+	// Add the custom OPENAI_BASE_URL parameter to accommodate the custom proxy address
+	OPENAI_BASE_URL(url?: string) {
+		if (!url) {
+			// https://github.com/openai/openai-node/blob/ed4219a565976750637d8c68b2b35409aca447af/src/index.ts#L103
+			return 'https://api.openai.com/v1';
+		}
+
+		parseAssert('OPENAI_BASE_URL', /^https?:\/\//.test(url), 'Must be a valid URL');
```

Comment:
You are requiring a domain with the protocol here, but the current https client being used does not accept `https://chat.example.com` as a "domain" (and rightfully so).